src/my_list.rb

```diff
@@ -0,0 +1,33 @@
+require_relative './enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*arr)
+    @list = arr
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] when creating the `each` method the intention is to do this without using the existing `each` method in ruby.

Reply:
@mikethreels updated !